frontend/public/components/RBAC/rules.jsx

```diff
@@ -4,27 +4,29 @@ import { connect } from 'react-redux';
 
 import { k8sPatch } from '../../module/k8s';
 import { RoleModel, ClusterRoleModel } from '../../models';
-import { Cog, ResourceIcon } from '../utils';
+import { Cog, EmptyBox, ResourceIcon } from '../utils';
 import { confirmModal } from '../modals';
 
-export const RulesList = ({rules, name, namespace}) => <div className="co-m-table-grid co-m-table-grid--bordered rbac-rules-list">
-  <div className="row co-m-table-grid__head">
-    <div className="col-xs-5 col-sm-4 col-md-3 col-lg-2">
-      Actions
+export const RulesList = ({rules, name, namespace}) => _.isEmpty(rules)
+  ? <EmptyBox label="Rule" />
```

Comment:
Nit:  it looks like we typically use the plural version of the label (e.g., Rules), but I notice we have one exception with ingress, which uses the singular "Rule".  Here and ingress should be "Rules"?